src/components/TopBar/TopBar.vue

```diff
@@ -46,6 +46,23 @@
 			</div>
 		</a>
 
+		<a v-if="showUpcomingEvent"
+			class="upcoming-event__card"
+			:aria-label="t('spreed', 'Next call')"
+			:href="nextEvent.calendarAppUrl"
+			:title="t('spreed', 'Click to open Calendar')"
```

Comment:
We shouldn't say anywhere "click to …" as especially titles can be read by screenreaders used by people who e.g. only use a keyboard.

Simply "Open Calendar" is enough.